package/src/skia/web/JsiSkCanvas.ts

```diff
@@ -374,4 +375,33 @@ export class JsiSkCanvas
   drawPicture(skp: SkPicture) {
     this.ref.drawPicture(JsiSkPicture.fromValue(skp));
   }
+
+  readPixels(
+    srcX: number,
+    srcY: number,
+    imageInfo: ImageInfo,
+    dest?: object,
```

Comment:
Should be: `dest?: Float32Array | Uint8Array` and we create the MallocObj inside:
https://github.com/google/skia/blob/d29cc3fe182f6e8a8539004a6a4ee8251677a6fd/modules/canvaskit/tests/core_test.js#L178C16-L178C79, then copy the typed array and free the MallocObj.

Reply:
readPixels always convert TypedArray.
So your think will make twice memory usage.